### ISSUE TITLE
how to install OMA-LWM2M-DevKit on Firefox？ 

### ISSUE THREAD

**Issue by zhanghe9702** [OpenMobileAlliance/OMA-LWM2M-DevKit#15]
the install instructions seems very outdate, and the addons can't be install correctly

**Comment by jpradocueva** [comment]
@zhanghe9702 yes, you are right DMSE has stopped updating the DevKit. I would suggest that you should raise this issue directly with DMSE at this link: https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/issues

The group may decide to update the DevKit if they hear that it is still being used by the industry. 
Thanks!
